scripts/huggingface/upload_model.py

```diff
@@ -0,0 +1,70 @@
+import inspect
+import logging
+import pathlib
+
+import hydra
+import torch
+from huggingface_hub import PyTorchModelHubMixin
+from hydra.utils import instantiate
+from omegaconf import DictConfig
+
+from the_well.data import WellDataModule
+
+logger = logging.getLogger("the_well")
+
+CONFIG_DIR = (
+    pathlib.Path(__file__) / ".." / ".." / ".." / "the_well" / "benchmark" / "configs"
```

Comment:
Isn't this better? pathlib's `/` operator overload is nice, but not always necessary :sweat_smile:

```suggestion
    pathlib.Path(__file__) / "../../../the_well/benchmark/configs"
```